<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidgetDocuments</class>
 <widget class="QTabWidget" name="TabWidgetDocuments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>507</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::East</enum>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <property name="iconSize">
   <size>
    <width>25</width>
    <height>25</height>
   </size>
  </property>
  <widget class="QWidget" name="T_taskDistro">
   <attribute name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/ic_insert_link_white_2x.png</normaloff>:/Icons/ic_insert_link_white_2x.png</iconset>
   </attribute>
   <attribute name="title">
    <string>توزيع الوثائق على الموظفين</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BT_associate">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
 position: relative;
    padding: 7px 12px;
        padding-top: 7px;
        padding-bottom: 7px;
    font-size: 15px;
    font-family: Ubuntu;
    color: #ffffff;
    white-space: nowrap;
    background-color: rgb(0, 163, 239);
   
    border-radius: 3px;
        border-top-left-radius: 3px;
        border-top-right-radius: 3px;
        border-bottom-right-radius: 3px;
        border-bottom-left-radius: 3px;
    border: 1px solid rgb(0, 163, 239);
       
    vertical-align: middle;
}

QPushButton:hover
{
background-color: rgb(11, 109, 255);
    border: 1px solid  rgb(11, 109, 255);
color: #ffffff;

}
</string>
          </property>
          <property name="text">
           <string>تعيين</string>
          </property>
          <property name="icon">
           <iconset resource="Icons.qrc">
            <normaloff>:/Icons/ic_insert_link_white_2x.png</normaloff>:/Icons/ic_insert_link_white_2x.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(42, 188, 255);</string>
          </property>
          <property name="text">
           <string>إختيار وثيقة / مذكرة :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CB_pickDoc">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(42, 188, 255);</string>
          </property>
          <property name="text">
           <string>إختيار موظف :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CB_pickEmpl">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QcomboBox
{
text-align:center;
}</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="maxCount">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:0px solid white;</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>681</width>
         <height>416</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="title">
           <string>الوثائق في طور الانجاز</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="topMargin">
            <number>15</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>تعديل الوثائق / الكتب</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QGroupBox" name="GB_updateClient">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupeBox
{
border:0px solid white;
border-top:1px solid #FF165A;
}</string>
      </property>
      <property name="title">
       <string>تعديل معلومات الوثائق</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetSearch" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget
{
	background-color:#D32F2F;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label_28">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="Icons.qrc">:/Icons/ic_search_white_2x.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_fullnameFiltre">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit
{
 margin-bottom: 2px;   
    padding: 7px 8px;
        padding-top: 7px;
        padding-bottom: 7px;
    outline: medium none;
        outline-width: medium;
        outline-style: none;
        outline-color: -moz-use-text-color;

    background-color: #C62828;
color:white;
    border: 1px solid #C62828;
      
    border-radius: 3px;
        border-top-left-radius: 3px;
        border-top-right-radius: 3px;
        border-bottom-right-radius: 3px;
        border-bottom-left-radius: 3px;
    vertical-align: middle;
}
QLineEdit:hover
{
	 border: 1px solid #C62828;
	 border-top-color: #C62828;
     border-bottom-color: #C62828;  

}</string>
            </property>
            <property name="maxLength">
             <number>30</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>ادخال اسم المذكرة او الكتاب المراد البحث عنه</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTableView" name="tableView">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QTableView
{
 margin-bottom: 2px;   
    padding: 7px 8px;
        padding-top: 7px;
        padding-bottom: 7px;
    outline: medium none;
        outline-width: medium;
        outline-style: none;
        outline-color: -moz-use-text-color;
    color: #333;
    background-color: #FFF;
    background-repeat: no-repeat;
    background-position: right center;
    border: 1px solid #c5c5c5;
        border-top-width: 1px;
        border-bottom-width: 1px;
        border-top-style: solid;
        border-bottom-style: solid;
        border-top-color: #c5c5c5;
        border-bottom-color: #c5c5c5;      
    border-radius: 3px;
        border-top-left-radius: 3px;
        border-top-right-radius: 3px;
        border-bottom-right-radius: 3px;
        border-bottom-left-radius: 3px;
    vertical-align: middle;
}
QTableView:hover
{
	 border: 1px solid #00bbff;
	 border-top-color: #00bbff;
        border-bottom-color: #00bbff;  

}</string>
           </property>
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::CustomDashLine</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QSpinBox" name="SB_idDocx">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maximum">
              <number>9999999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
               <kerning>false</kerning>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{

	color: rgb(0, 163, 239);
}</string>
             </property>
             <property name="text">
              <string>يوم التسليم</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="DE_deliveryDayx">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
               <kerning>false</kerning>
              </font>
             </property>
             <property name="currentSection">
              <enum>QDateTimeEdit::YearSection</enum>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
               <kerning>false</kerning>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{

	color: rgb(0, 163, 239);
}</string>
             </property>
             <property name="text">
              <string>عدد الصفحات المنجزة</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SB_pagesDonex">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="maximum">
              <number>5000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SB_totalPagesx">
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <number>5000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
               <kerning>false</kerning>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{

	color: rgb(0, 163, 239);
}</string>
             </property>
             <property name="text">
              <string>الطباعة</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CB_isPrintedx">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="maxCount">
              <number>3</number>
             </property>
             <item>
              <property name="text">
               <string>لم تطبع بعد</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>تمت الطباعة</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="LY_updateClient">
             <item>
              <widget class="QPushButton" name="BT_updateDoc">
               <property name="styleSheet">
                <string notr="true">QPushButton
{
 position: relative;
    padding: 7px 12px;
        padding-top: 7px;
        padding-bottom: 7px;
    font-size: 15px;
    font-family: Ubuntu;
    color: #ffffff;
    white-space: nowrap;
    background-color: rgb(0, 163, 239);
   
    border-radius: 3px;
        border-top-left-radius: 3px;
        border-top-right-radius: 3px;
        border-bottom-right-radius: 3px;
        border-bottom-left-radius: 3px;
    border: 1px solid rgb(0, 163, 239);
       
    vertical-align: middle;
}

QPushButton:hover
{
background-color: rgb(11, 109, 255);
    border: 1px solid  rgb(11, 109, 255);
color: #ffffff;

}
</string>
               </property>
               <property name="text">
                <string>حفظ</string>
               </property>
               <property name="icon">
                <iconset resource="Icons.qrc">
                 <normaloff>:/Icons/ic_save_white_2x.png</normaloff>:/Icons/ic_save_white_2x.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="shortcut">
                <string>Return, Enter</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_10">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSplitter" name="splitter_11">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="splitter_12">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="T_progressLevels">
   <attribute name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/ic_archive_white_2x.png</normaloff>:/Icons/ic_archive_white_2x.png</iconset>
   </attribute>
   <attribute name="title">
    <string>الارشيف</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QScrollArea" name="scrollArea_archive">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:0px solid white;</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>681</width>
         <height>483</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="title">
           <string>الوثائق التي تم انجازها</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="topMargin">
            <number>15</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
